include/misc/print_table.hpp

```diff
@@ -0,0 +1,384 @@
+/*
+    Header file from: https://github.com/friedmud/variadic_table/blob/master/include/VariadicTable.h
+    Source project: https://github.com/friedmud/variadic_table
+*/
```

Comment:
where is this used? If it is only in examples then maybe put it there instead of `misc` ?

Reply:
It is not examples. We use it to print the MCMC diagnostics at the tests.

Reply:
then why not including it in tests ?